### PR TITLE
Feature/feelings component

### PR DESCRIPTION
**Pull request by jfogiato**
### Files Changed: 
`reportWebVitals` (removed), `index.js` (removed webVitals), `App.css`, `App.js`, `Feelings.js`, `Feelings.css` 

### What Has Changed: 
Added initial feelings "buttons" (three for each piece of the center of the [wheel](https://static1.squarespace.com/static/57b5ef68c534a5cc06edc769/t/5ea76681f6730e409abf228b/1588030494465/Feelings+Wheel.pdf) - can def adjust as needed), add methods to `App.js` to update state from click of feelings button, initial styling of `Feelings.js`, added global CSS variables

### Notes: 
Nothing major, just very open to changing anything and everything styling-wise! And also any refactoring that you see!!

### Closes: 
None

### CODE REVIEW

File: src/Components/App/App.css [hunk 1]
```diff
@@ -1,38 +1,7 @@
...
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
```
Comment (espressoGoddess) [inline src/Components/App/App.css:37]: thanks for getting all the ickiness outta here!

File: src/Components/App/App.css [hunk 2]
```diff
@@ -1,38 +1,7 @@
...
+:root {
+  --primary-bkgd-color: rgb(255, 204, 0);
```
Comment (espressoGoddess) [inline src/Components/App/App.css:2]: i am so pumped to use these variable to streamline our colors!

File: src/Components/App/App.js [hunk 3]
```diff
@@ -18,10 +18,19 @@ const App = () => {
   const [activities, setActivities] = useState(activityData);
   const [filteredActivities, setFilteredActivities] = useState([]);
 
+  useEffect(() => {
```
Comment (espressoGoddess) [inline src/Components/App/App.js:21]: LOVE HOW YOU SET THIS UP TO BE A SEAMLESS TRANSITION TO FETCHING OUR DATA

File: src/Components/Feelings/Feelings.js [hunk 4]
```diff
@@ -1,14 +1,40 @@
-import React from "react";
+import React from 'react';
```
Comment (espressoGoddess) [inline src/Components/Feelings/Feelings.js:1]: nice attention to detail

File: src/Components/Feelings/Feelings.js [hunk 5]
```diff
@@ -1,14 +1,40 @@
...
-      <h2>How are you Feeling?</h2>
-      <p>Happy</p>
-      <p>Sad</p>
-      <p>Mad</p>
-    </>
+    <section className='feelings-page'>
+      <h1>BEAM</h1>
+      <h2 className='feelings-header'>How are you today?</h2>
+      <section className='feelings-container'>
+        <p className='feeling happy' onClick={callBack}>Happy</p>
```
Comment (espressoGoddess) [inline src/Components/Feelings/Feelings.js:15]: i like how you added a placeholder call back function!

**Review by espressoGoddess — APPROVED** [review]

File: src/Components/Feelings/Feelings.js (hunk 5 above)
Comment (chrissycooper) [reply src/Components/Feelings/Feelings.js:15]: agree! But, I'm not sure it's very semantic to name it `callBack`

File: src/index.js [hunk 6]
```diff
@@ -2,16 +2,10 @@ import React from 'react';
...
 import App from './Components/App/App';
-import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>
 );
-
```
Comment (chrissycooper) [inline src/index.js:13]: thx for doing clean up

File: src/Components/Feelings/Feelings.js (hunk 5 above)
Comment (jfogiato) [reply src/Components/Feelings/Feelings.js:15]: Ah yeah I figured that having one function up top was more readable than having it in every single line 😅

File: src/Components/App/App.js (hunk 3 above)
Comment (jfogiato) [reply src/Components/App/App.js:21]: I'm 'hook' d

File: src/Components/Feelings/Feelings.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-content: flex-start;
+  justify-content: center;
+}
+
+.feeling{
+  background-color: var(--tertiary-bkgd-color);
+  padding: 1vmin 2vmin 1vmin 2vmin;
```
Comment (chrissycooper) [inline src/Components/Feelings/Feelings.css:26]: interesting use of vmin! I haven't tried this responsive unit out yet, I'm into it

**Review by chrissycooper — APPROVED** [review]

File: src/Components/App/App.css [hunk 8]
```diff
@@ -1,38 +1,7 @@
...
+  --secondary-bkgd-color: rgb(254, 84, 84);
+  --tertiary-bkgd-color: rgb(230, 229, 229);
+  --hover-bkgd-color: ivory;
+  --bkgd-radial-gradient: content-box radial-gradient(var(--secondary-bkgd-color), var(--primary-bkgd-color));
```
[the remark this answers is not in the document]
Comment (jfogiato) [reply src/Components/App/App.css:6]: Happy to check it out with ya @chrissycooper !!

File: src/Components/Feelings/Feelings.js [hunk 9]
```diff
@@ -1,14 +1,40 @@
-import React from "react";
+import React from 'react';
 import './Feelings.css'
 
-const Feelings = () => {
+const Feelings = ({ setFeeling }) => {
+
+  const callBack = ({ target }) => {
```
[the remark this answers is not in the document]
Comment (jfogiato) [reply src/Components/Feelings/Feelings.js:6]: 🤌

File: src/Components/Feelings/Feelings.js (hunk 5 above)
Comment (jfogiato) [reply src/Components/Feelings/Feelings.js:15]: @chrissycooper Super fair point - I'm going to adjust and push up the change so when @KatherineBlaine checks it out & merges, that will be fixed!

File: src/index.js (hunk 6 above)
Comment (jfogiato) [reply src/index.js:13]: 🫡

File: src/Components/Feelings/Feelings.js [hunk 10]
```diff
@@ -12,30 +12,30 @@ const Feelings = ({ setFeeling }) => {
...
-        <p className='feeling fearful' onClick={callBack}>Fearful</p>
-        <p className='feeling fearful' onClick={callBack}>Anxious</p>
-        <p className='feeling fearful' onClick={callBack}>Insecure</p>
-        <p className='feeling angry' onClick={callBack}>Angry</p>
-        <p className='feeling angry' onClick={callBack}>Distant</p>
-        <p className='feeling angry' onClick={callBack}>Frustrated</p>
-        <p className='feeling disguisted' onClick={callBack}>Disguisted</p>
-        <p className='feeling disguisted' onClick={callBack}>Disappointed</p>
-        <p className='feeling disguisted' onClick={callBack}>Awful</p>
+        <p className='feeling happy' onClick={selectFeeling}>Happy</p>
```
Comment (KatherineBlaine) [inline src/Components/Feelings/Feelings.js:15]: Love that you put the 'root feeling' as the class name. Super smart way to organize them.

File: src/Components/App/App.css (hunk 8 above)
[the remark this answers is not in the document]
Comment (KatherineBlaine) [reply src/Components/App/App.css:6]: gradient 😍

File: src/Components/App/App.css (hunk 1 above)
Comment (KatherineBlaine) [reply src/Components/App/App.css:37]: Good riddance boilerplate

**Review by KatherineBlaine — APPROVED** [review]

**Pull request merged**